internal/pkg/service/buffer/sink/tablesink/storage/repository/slice_repo_test.go

```diff
@@ -91,7 +91,10 @@ func TestRepository_Slice(t *testing.T) {
 	{
 		fileKey := test.NewFileKey()
 		if err := r.Create(fileKey, volumeID, 0).Do(ctx).Err(); assert.Error(t, err) {
-			assert.Equal(t, `sink "123/456/my-source/my-sink" not found in the source`, err.Error())
+			assert.Equal(t, strings.TrimSpace(`
+- sink "123/456/my-source/my-sink" not found in the source
+- file "123/456/my-source/my-sink/2000-01-01T19:00:00.000Z" not found in the sink
+`), err.Error())
```

Comment:
:one: The result of simplified code, there are now two error.

Tt is an unexpected state, so it is not problem, in addition, it can help with debugging when we know that the file does not even exist.